express-static/index.js

```diff
@@ -0,0 +1,14 @@
+const express = require('express');
+const path = require('path');
+
+const app = express();
+
+const joined = path.join('public', __dirname);
+console.log(joined);
+
+app.use(express.static('public'));
+
```

Comment:
Use your joined path here instead of just `public`. All file system operations are relative to `process.cwd()` unless they are given an absolute path.